go-controller/pkg/ovn/namespace.go

```diff
@@ -314,6 +314,8 @@ func (oc *Controller) updateNamespace(old, newer *kapi.Namespace) {
 		// deny rules are all one per namespace
 		if err := oc.setACLDenyLogging(old.Name, nsInfo, nsInfo.aclLogging.Deny); err != nil {
 			klog.Warningf(err.Error())
+		} else if err := oc.setNamespaceACLLogging(old.Name, nsInfo, nsInfo.aclLogging.Allow); err != nil {
```

Comment:
I would consider condensing `setACLDenyLogging` and `setNamespaceACLLogging` into a single function